documentation/docs/pmm-upgrade/external_postgres_pmm_upgrade.md

```diff
@@ -48,117 +49,115 @@ Depending on your deployment method, use one of the following approaches:
+### 4. Upgrade to PMM 3.2.0 and restart PMM Server with the new configuration
 
 Modify your startup command or configuration file to use the new parameters:
+!!! note "Version Upgrade"
+    This step will upgrade your PMM instance to version 3.2.0 and apply the new database configuration. 
+
 
 === "Docker"
-    ```bash
-    docker run -d \
-      -p 443:443 \
-      -v pmm-data:/srv \
-      -e GF_DATABASE_TYPE=postgresql \
-      -e GF_DATABASE_USER=your_user \
-      -e GF_DATABASE_PASSWORD=your_password \
-      -e GF_DATABASE_HOST=your_host:your_port \
-      -e GF_DATABASE_NAME=your_db_name \
-      -e PMM_DISABLE_BUILTIN_POSTGRES=1 \
-      --name pmm-server \
-      percona/pmm-server:3.2.0
-    ```
+   ```bash
+   docker run -d \
+     -p 443:443 \
+     -v pmm-data:/srv \
+     -e GF_DATABASE_USER=your_user \
+     -e GF_DATABASE_PASSWORD=your_password \
+     -e GF_DATABASE_HOST=your_host:your_port \
+     -e GF_DATABASE_NAME=your_db_name \
+     -e PMM_DISABLE_BUILTIN_POSTGRES=1 \
+     --name pmm-server \
+     percona/pmm-server:3.2.0
+   ```
 
 === "Podman"
-    ```bash
-    podman run -d \
-      -p 443:443 \
-      -v pmm-data:/srv \
-      -e GF_DATABASE_TYPE=postgresql \
-      -e GF_DATABASE_USER=your_user \
-      -e GF_DATABASE_PASSWORD=your_password \
-      -e GF_DATABASE_HOST=your_host:your_port \
-      -e GF_DATABASE_NAME=your_db_name \
-      -e PMM_DISABLE_BUILTIN_POSTGRES=1 \
-      --name pmm-server \
-      percona/pmm-server:3.2.0
-    ```
+   ```bash
+   podman run -d \
+     -p 443:443 \
+     -v pmm-data:/srv \
+     -e GF_DATABASE_USER=your_user \
+     -e GF_DATABASE_PASSWORD=your_password \
+     -e GF_DATABASE_HOST=your_host:your_port \
+     -e GF_DATABASE_NAME=your_db_name \
+     -e PMM_DISABLE_BUILTIN_POSTGRES=1 \
+     --name pmm-server \
+     percona/pmm-server:3.2.0
+   ```
 
 === "Docker Compose"
-    Update your `docker-compose.yml` file:
-    ```yaml
-    services:
-      pmm-server:
-        image: percona/pmm-server:3.2.0
-        ports:
-          - "443:443"
-        volumes:
-          - pmm-data:/srv
-        environment:
-          - GF_DATABASE_TYPE=postgresql
-          - GF_DATABASE_USER=your_user
-          - GF_DATABASE_PASSWORD=your_password
-          - GF_DATABASE_HOST=your_host:your_port
-          - GF_DATABASE_NAME=your_db_name
-          - PMM_DISABLE_BUILTIN_POSTGRES=1
-        restart: always
-    
-    volumes:
-      pmm-data:
-    ```
-    Then restart:
-    ```bash
-    docker-compose up -d
-    ```
+   Update your `docker-compose.yml` file:
+   ```yaml
+   services:
+     pmm-server:
+       image: percona/pmm-server:3.2.0
+       ports:
+         - "443:443"
+       volumes:
+         - pmm-data:/srv
+       environment:
+         - GF_DATABASE_USER=your_user
+         - GF_DATABASE_PASSWORD=your_password
+         - GF_DATABASE_HOST=your_host:your_port
+         - GF_DATABASE_NAME=your_db_name
+         - PMM_DISABLE_BUILTIN_POSTGRES=1
```

Comment:
The variable we provide here are insufficient to run PM with an external database. The reason is that PMM does not rely on `GF_` variable.

There should be more:

- PMM_POSTGRES_ADDR (i.e. host:port)
- PMM_POSTGRES_DBNAME
- PMM_POSTGRES_USERNAME
- PMM_POSTGRES_DBPASSWORD

IMO, we should additionally refer to this [section](https://docs.percona.com/percona-monitoring-and-management/3/reference/third-party/postgresql.html) here.